api_app/playbooks_manager/migrations/0027_add_phoneinfoga_scan_to_free_to_use_analyzers.py

```diff
@@ -0,0 +1,34 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+
+from django.db import migrations
+
+
+def migrate(apps, schema_editor):
+    playbook_config = apps.get_model("playbooks_manager", "PlaybookConfig")
+    AnalyzerConfig = apps.get_model("analyzers_manager", "AnalyzerConfig")
+    pc = playbook_config.objects.get(name="FREE_TO_USE_ANALYZERS")
+    pc.analyzers.add(AnalyzerConfig.objects.get(name="Phoneinfoga").id)
+    pc.full_clean()
+    pc.save()
+
+
+def reverse_migrate(apps, schema_editor):
+    playbook_config = apps.get_model("playbooks_manager", "PlaybookConfig")
+    AnalyzerConfig = apps.get_model("analyzers_manager", "AnalyzerConfig")
+    pc = playbook_config.objects.get(name="FREE_TO_USE_ANALYZERS")
+    pc.analyzers.remove(AnalyzerConfig.objects.get(name="Phoneinfoga").id)
+    pc.full_clean()
+    pc.save()
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("playbooks_manager", "0026_add_zippy_scan_to_free_to_use"),
+        ("analyzers_manager", "0065_analyzer_config_phoneinfoga"),
+    ]
+
+    operations = [
+        migrations.RunPython(migrate, reverse_migrate),
+    ]
```

Comment:
This is not required because this analyzer is "optional" and it requires to run an additional container.
To avoid to have it providing errors in the base installation, that is the most commonly used, I would not add it in the default playbook. Please remove.

Reply:
noted. Shouldn't we also mention that such type of analyzers which are free but `docker-based`/`optional ` can't go in default playbook in the `docs` or the `PR checklist`; to avoid such implementations in future.

Reply:
sure it would make sense to explicitly say it in the PR template

Reply:
just one more concern, about another docker based analyzer, cyberchef which is in the default playbook
![image](https://github.com/intelowlproject/IntelOwl/assets/102874321/d003186d-3913-42f0-b25f-2bfe11f9c227)